presto/scripts/run_benchmarks.sh

```diff
@@ -0,0 +1,273 @@
+	    echo_error "Query $query timed out after ${timeout_seconds}s"
+            rc=1
+            break
+        fi
+        final_response=$(curl -sS "$current_uri")
+        state=$(echo "$final_response" | jq -r '.stats.state // empty')
+        if [[ "$state" == "FAILED" ]]; then
+	    local error=$(echo "$final_response" | jq -r '.error.message // "Unknown error"')
+	    echo_error "Query $query failed: $error"
+            rc=1
+	    break
+        fi
+        [[ "$state" == "FINISHED" ]] && break
+        local next_uri_response=$(echo "$final_response" | jq -r '.nextUri // empty')
+        [[ -n "$next_uri_response" ]] && current_uri="$next_uri_response"
+    done
+    echo "$final_response"
+    return $rc
+}
+
+function filter_output() {
+    local query=$1
+    local execution_time=$2
+    local final_response=$3
+    if [[ -n $final_response ]]; then
+        local stats=$(echo "$final_response" | jq '.stats // {}')
+        local processed_rows=$(echo "$stats" | jq -r '.processedRows // 0')
+        local processed_bytes=$(echo "$stats" | jq -r '.processedBytes // 0')
+        local cpu_time_ms=$(echo "$stats" | jq -r '.cpuTimeMillis // 0')
+        local wall_time_ms=$(echo "$stats" | jq -r '.wallTimeMillis // 0')
+        local elapsed_time_ms=$(echo "$stats" | jq -r '.elapsedTimeMillis // 0')
+        jq -C -n \
+           --arg query "$query" \
+           --arg execution_time "$execution_time" \
+           --arg processed_rows "$processed_rows" \
+           --arg processed_bytes "$processed_bytes" \
+           --arg cpu_time_ms "$cpu_time_ms" \
+           --arg wall_time_ms "$wall_time_ms" \
+           --arg elapsed_time_ms "$elapsed_time_ms" \
+           '{
+                query_number: $query,
+		curl_execution_time_seconds: ($execution_time | tonumber),
+		processed_rows: ($processed_rows | tonumber),
+		processed_bytes: ($processed_bytes | tonumber),
+		cpu_time_ms: ($cpu_time_ms | tonumber),
+		wall_time_ms: ($wall_time_ms | tonumber),
+		elapsed_time_ms: ($elapsed_time_ms | tonumber)
+           }'
+    else
+        jq -C -n \
+           --arg query "$query" \
+           --arg execution_time "$execution_time" \
+           '{
+                query_number: $query,
+		execution_time_seconds: ($execution_time | tonumber)
+            }'
+    fi
+}
+
+function create_tables() {
+    presto_cli "CREATE SCHEMA IF NOT EXISTS hive.$SCHEMA"
+    local pattern="\/([^\/]*)\.sql"
+    for sql_file in $(ls $BASE_DIR/presto/testing/integration_tests/schemas/tpch/*.sql); do
+        local drop_table=""
+	if [[ "$sql_file" =~ $pattern ]]; then
+	    local table_name="${BASH_REMATCH[1]}"
+	    drop_table="DROP TABLE IF EXISTS $table_name"
+	else
+	    echo "failed to parse schema files"
+	    exit 1
+	fi
+        presto_cli "$drop_table"
+        local table_dir="/var/lib/presto/data/hive/data/integration_test/tpch/$table_name"
+        local create_table=$(cat $sql_file | sed "s+{file_path}+$table_dir+g")
```

Comment:
please make the table dir configurable to register existing data (for example existing SF100 data in a directory)